### ISSUE TITLE
How to change DataDog java agent trace api version to V0.5

### ISSUE THREAD

**Issue by peachisai** [DataDog/dd-trace-java#7277]
Hi, I would like to ask some questions.
I am testing how to send DataDog Java agent trace to my local terminal. But the default trace version is v0.4, and now I want to change the api version to v0.5 or v0.3.

My local system is Windows.

I had read the official documentation, and I have tried to use the JVM arguments:
```
-Ddd.trace.api.version=v0.5
```
or config the environment variable
```
 DD_TRACE_API_VERSION v0.5
```
Both approaches are ineffective. It was still v0.4 api version.
```
dd-java-agent version 1.35.2
```

Is there anything  else I need to configure?

**Comment by mcculls** [comment]
Hi @peachisai, the Java tracer does not currently support `DD_TRACE_API_VERSION`

To allow it to use the v0.5 protocol you need to add this JVM option:
```
-Ddd.trace.agent.v0.5.enabled=true
```
or alternatively set this environment variable:
```
DD_TRACE_AGENT_V0_5_ENABLED=true
```

Note there has been a previous [issue](https://github.com/DataDog/dd-trace-java/issues/5314) reported when using the v0.5 protocol with the Java tracer which was fixed by reverting to v0.4.  This was likely specific to the customers environment, but wanted to make you aware.

**Comment by peachisai** [comment]
> Hi @peachisai, the Java tracer does not currently support `DD_TRACE_API_VERSION`
> 
> To allow it to use the v0.5 protocol you need to add this JVM option:
> 
> ```
> -Ddd.trace.agent.v0.5.enabled=true
> ```
> 
> or alternatively set this environment variable:
> 
> ```
> DD_TRACE_AGENT_V0_5_ENABLED=true
> ```
> 
> Note there has been a previous [issue](https://github.com/DataDog/dd-trace-java/issues/5314) reported when using the v0.5 protocol with the Java tracer which was fixed by reverting to v0.4. This was likely specific to the customers environment, but wanted to make you aware.

It works , Thank you very much!
